Demos/CosmosGraphicSubsystem/Kernel.cs

```diff
@@ -79,6 +84,11 @@ protected override void Run()
                 pen.Color = Color.Chocolate;
```

Comment:
I don't understand this change maybe VmWare supports 16 bit color, but surely not VBE at least until we do not do the conversion from RGB32 to RGB16.
In theory this mode change should throw for VBE as it is not supported.

Reply:
I've reverted it in the latest commits, I must've got confused with the deployment profiles
32-bit works fine in both VMWare and Bochs, but only 16-bit with VMWare, and throwing as necessary

Reply:
For 800x600 anyway - for 1024x768, that needs checking